### DISCUSSION TITLE
Public domain media not using the Creative Commons Public Domain Mark

### DISCUSSION THREAD

**Discussion by zackkrida** [WordPress/openverse#89] [General]
Currently, Openverse does _not_ include public domain works unless they use the Public Domain Mark `PDM` tool from Creative Commons[^1]. This is because without PDM, a work may be:

- in the 'public domain'
- have 'no known copyright restrictions'

in one jurisdiction, but _could_ in others. Here are CC's recommendations for when _not_ to use `PDM` in their entirety[^2]:

> The Public Domain Mark is recommended for works that are free of known copyright around the world. These will typically be very old works.  It is not recommended for use with works that are in the public domain in some jurisdictions if they also known to be restricted by copyright in others.
>
> A work may have limited or “hybrid” public domain status for several reasons.  Some jurisdictions have unusually long copyright terms, which may mean that a work free from copyright restrictions most everywhere in the world could still be protected by the copyright laws of that particular country.  Sometimes a work is no longer restricted by copyright in a jurisdiction because the author or owner failed to comply with local formalities such as renewal, where those formalities apply.  It could also be the case for works that are deemed not protected by copyright by operation of law in a particular jurisdiction, but that are afforded protection under the copyright laws of other jurisdictions.
>
> CC does not recommend the Public Domain Mark for works with limited, hybrid public domain status at this time.

We need to decide if we want to continue this practice or support these works.

[^1]: [Public Domain Mark 1.0](https://creativecommons.org/publicdomain/mark/1.0/)
[^2]: [PDM Guidelines](https://creativecommons.org/share-your-work/public-domain/pdm/)

**Comment by obulat** [comment]
I am in favor of using Public domain media even without a PDM or CC0 marks. There is currently a large body of very high-quality media that are published on Wikimedia and Flickr by GLAM[^1] institutions that do not have PDM or CC0. And I believe we will serve our users better if we do include this type of media.

It might be a good idea to consult with lawyers, though. Or at least to see what these providers give as justification for including non-PDM or CC0 works in their collections.

[^1]: Galleries, Libraries, Archives and Museums

**Reply by obulat** [reply]
Interesting statistics from the [Flickr Commons Revitalization: Research Report](https://docs.google.com/document/d/19eHlYDCJS2F4KH-Eml8mW9aVqP5WVOaKN_-5XFW4-wc) about the licenses used by the 114 cultural institutions in 24 countries around the world, who share historical photography on Flickr.

> Even though the core usage assertion in the program is that images are shared with &ldquo;no known copyright restriction,&rdquo; we did uncover that there is some licensed content as well. Here&rsquo;s the breakdown, as of August 2021:

All rights reserved | 15,792
-- | --
CC-BY-NC-SA | 13
CC-BY-NC | 2
CC-BY-NC-ND | 38,331
CC-BY | 560
No known copyright restrictions (NKCR) | 7,038,049
CC0 | 528
Public Domain Mark | 10,331

We are not ingesting the overwhelming majority of these media items.

**Comment by zackkrida** [comment]
The fact that Flickr is comfortable showing these images makes me less worried about legal implications. I think this is more of a philosophical questions, if we want to 'support'/'endorse' the non-PDM public domain, whatever that is lol.

**Reply by fcoveram** [reply]
Agree with this. I think we can display a message saying something like _"this work has not been marked as PDM nor CC0 but it belongs to the public domain in some jurisdictions"_ and link to a page that details the legal implications.

From the value proposition perspective, we _do want_ to boost the use of PDM content. The best experience for its correct use is the work we that need to do.

**Reply by obulat** [reply]
I think that displaying a message like @panchovm proposes on non-PDM and non-CC0 items is a way we could show our support/endorsements for these tools, while also providing our users access to the media which is most likely Public Domain.

**Comment by krysal** [comment]
I'm really interested in knowing how works are classified in "No Known Copyright Restrictions" (NKCR) as this is key in determining in which sense they are open, seems to be mostly a legacy term to encapsulate works with unknown or uncommon licenses, in order to be included in collections as we intend to do here.

The same Flickr report says:
> Lots of cultural organizations have embraced the open-content movement and have moved past the NKCR assertion to make their digital collections available with explicit open licenses like Creative Commons.

> NKCR approach is dated now

Given the nature of copyright, each work has all rights reserved unless there is a license in which the author explicitly releases them. Some of these works might have less common non-CC licenses, which case makes me wonder: will we provide instructions on how to cite/use such works? I think we should do it, bring clarity instead of keep carrying confusion. This is where we would need a legal person dedicated to analyze licenses and their permissions.
 
If a work is confirmed to be of Public Domain (basically means no rights reserved, zero!) in one jurisdiction then I think we can include it, as it means the author(s) are willing to cede the rights (or most probably they are dead a long time ago) and the final "mark" as the public domain is mostly a bureaucracy thing, but if the license of the work is unknown I definitely believe it should be **not included** until its licensing is clarified.

**Reply by zackkrida** [reply]
Yes, it's important to note that even though Flickr Commons _does currently_ use NKCR, they intend to move away from it in favor of CC0 and PDM

**Reply by fcoveram** [reply]
It makes sense to now show unknown license work as **we do not know** "author's intention when releasing their work."
